testing/test_testingbot.py

```diff
@@ -95,5 +95,9 @@ def test_invalid_host(failure, monkeypatch, tmpdir):
     ("protocol", "host", "port"), [("http", "localhost", "4445"), ("https", HOST, PORT)]
 )
 def test_executor_url(protocol, host, port):
+    # Import needs to be here, see here:
+    # https://github.com/pytest-dev/pytest/issues/5330#issuecomment-497146362
+    from pytest_selenium.drivers.testingbot import TestingBot
```

Comment:
I'm wondering if we can use `... TestingBot as TB`

Reply:
I can try.

Reply:
Fixed.